lua/gruvbox-baby/theme.lua

```diff
@@ -381,6 +381,18 @@ function M.setup(config)
     })
   end
 
+  if config.hideInactiveStatusline then
+    local inactive = { style = "underline", bg = c.background_colored, fg = c.foreground, sp = c.foreground }
```

Comment:
Could you change `background_colored` here and remove those 2 `background_colored` fields from the palette? I feel they don't serve a purpose